homeassistant/components/config/config_entries.py

```diff
@@ -372,6 +372,30 @@ async def ignore_config_flow(hass, connection, msg):
     connection.send_result(msg["id"])
 
 
+@websocket_api.require_admin
+@websocket_api.async_response
```

Comment:
This needs to go below the schema to work correctly with typing.
```suggestion
```

Reply:
How come mypy does not complain, is `components/config` not typed?

Reply:
https://github.com/home-assistant/core/blob/567f07c96eedfce739ecfff79314c6a2d582cabc/script/hassfest/mypy_config.py#L25